build.gradle

```diff
@@ -123,3 +123,10 @@ configure(javaProjects()) {
 }
 
 apply from: file("${rootProject.projectDir}/tools/gradle/publish.gradle")
+tasks.register('formatPython') {
+    doFirst {
+        exec {
+            commandLine "bash", "-c", "find . -name '*.py' -print0 | xargs -0 yapf --in-place"
```

Comment:
yapf is not installed by default

Reply:
That's why this is not enforced, developer has to install it manually